configuration/config-txt/pi4-hdmi.md

```diff
@@ -4,13 +4,13 @@ In order to support dual displays, and modes up to 4k60, the Raspberry Pi 4 has
 
 Every HDMI mode has a list of timings that control all the parameters around sync pulse durations. These are typically defined via a pixel clock, and then a number of active pixels, a front porch, sync pulse, and back porch for each of the horizontal and vertical directions. 
 
-Running everything at 2 pixels per clock means that the Pi4 can not support a timing where any of the horizontal timings are not divisible by 2. The firmware and Linux kernel will filter out any mode that does not fulfill this criteria.
+Running everything at 2 pixels per clock means that the Pi4 can not support a timing where _any_ of the horizontal timings are not divisible by 2. The firmware and Linux kernel will filter out any mode that does not fulfill this criteria.
 
-There is only one mode in the CEA and DMT standards that falls into this category - DMT mode 81, which is 1366x768 @ 60Hz. 1366x768 is already an unusual mode as normally the width of modes is divisible by 8, whereas 1366 is not.
+There is only one mode in the CEA and DMT standards that falls into this category - DMT mode 81, which is 1366x768 @ 60Hz (and has odd-numbered horizontal sync and back porch timings). 1366x768 is also an unusual mode for having a width that isn't divisible by 8.
```

Comment:
The brackets seem funny. 

...into this category - DMT mode 81, which is 1366x768 @ 60Hz. This mode has odd values for the horizontal sync and back porch timings. It's also an unusual mode for having a width that isn't divisible by 8.